xml/System.Drawing.Design/IconEditor.xml

```diff
@@ -124,7 +124,7 @@
         <Parameter Name="e" Type="System.Drawing.Design.IconEditor" Index="0" FrameworkAlternate="netframework-1.1;netframework-2.0;netframework-3.0;netframework-3.5;netframework-4.0;netframework-4.5;netframework-4.5.1;netframework-4.5.2;netframework-4.6;netframework-4.6.1;netframework-4.6.2;netframework-4.7;netframework-4.7.1;netframework-4.7.2;netframework-4.8" />
       </Parameters>
       <Docs>
-        <param name="editor">To be added.</param>
+        <param name="editor">Specifies an editor.</param>
```

Comment:
This should be the same description as the other one. The information above shows for which platforms this param name is valid.

```suggestion
        <param name="editor">The <see cref="T:System.Drawing.Design.IconEditor" /> for which the filter string will be created.</param>
```